diff --git a/node_modules/@react-native/gradle-plugin/settings.gradle.kts b/node_modules/@react-native/gradle-plugin/settings.gradle.kts
index b3c46a3..c8e8e87 100644
--- a/node_modules/@react-native/gradle-plugin/settings.gradle.kts
+++ b/node_modules/@react-native/gradle-plugin/settings.gradle.kts
@@ -7,14 +7,11 @@
 
 pluginManagement {
   repositories {
-    mavenCentral()
+    maven { url = uri("https://maven-central.storage-download.googleapis.com/maven2/") }
     google()
-    gradlePluginPortal()
   }
 }
 
-plugins { id("org.gradle.toolchains.foojay-resolver-convention").version("0.5.0") }
-
 include(
     ":react-native-gradle-plugin",
     ":settings-plugin",
